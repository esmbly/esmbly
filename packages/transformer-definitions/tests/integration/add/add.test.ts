import { testRunner } from '../../helpers/testRunner';

jest.mock('@esmbly/printer');

const program = `
  export function add(a: number, b: number): number {
    return a + b;
  }
`;

describe('integration: add', () => {
  it('generates a TypeScript definition file', async () => {
    const [{ content }] = await testRunner(program);
    expect(content).toMatchSnapshot();
  });
});
