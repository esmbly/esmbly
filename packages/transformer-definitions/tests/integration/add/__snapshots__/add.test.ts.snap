// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`integration: add generates a TypeScript definition file 1`] = `
"export declare function add(a: number, b: number): number;
"
`;
